client/apps/edit/components/edit_container.jsx

```diff
@@ -48,7 +48,7 @@ export class EditContainer extends Component {
       lastUpdated: null,
       isOtherUserInSession: !!props.currentSession && !isCurrentUserEditing,
       inactivityPeriodEntered: false,
-      shouldShowModal: true,
+      shouldShowModal: !props.article.isNew(),
```

Comment:
@l2succes - Since this is a bug could possibly use a test

Reply:
Yes for sure. I'll add tests in the actual PR. Just wanted to get this out to unblock the editor team